lib/BuildSystem/SwiftTools.cpp

```diff
@@ -353,6 +381,15 @@ class SwiftCompilerShellCommand : public ExternalCommand {
     SmallString<16> masterDepsPath;
     llvm::sys::path::append(masterDepsPath, tempsPath, "master.swiftdeps");
     os << "  \"\": {\n";
+    if (enableWholeModuleOptimization) {
```

Comment:
we need depsPath in both cases
object is not needed for !parallel-WMO == true but it can just stay there